readthedocs/starterguides/userguides/object_lifecycle_management_userguide.md

```diff
@@ -0,0 +1,89 @@
+# Opensds Object Lifecycle Management: User Guide
+
+## Introduction to Object lifecycle
+OpenSDS is aimed at addressing the storage integration challenges of both the Cloud Native environment and traditional IT environment. OpenSDS multi-cloud allows user to manage distributed cloud environment on a single platform. Object lifecycle management feature gives freedom to user to manage buckets and objects in the cloud using OpenSDS interface.
+
+## Getting Started!
+This guide is a tutorial which shows how to install, configure, and use Object lifecycle management feature in a simple OpenSDS setup.
+By following this guide you will be able to install OpenSDS multi-cloud with creating backends and buckets, uploading object and setting up the lifecycle for those objects.
+
+## Installing Opensds multi-cloud
+
+**Download opensds-installer code**
```

Comment:
Use ### for sub-heading instead of **bold**

Reply:
Updated